<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-FB27E98" name="test01.Timeout" tests="1" failures="0" timestamp="2 Jan 2019 03:12:52 GMT" time="2.016" errors="1">
  <testcase name="cs" time="2.016" classname="test01.Timeout">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.cs() didn&#039;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.cs() didn't finish within the time-out 2000
	at java.util.concurrent.ThreadPoolExecutor.processWorkerExit(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- cs -->
</testsuite> <!-- test01.Timeout -->
